Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM quay.io/ukhomeofficedigital/nodejs-base:v6.9.1
+
+RUN yum clean all && \
+  yum update -y -q && \
+  yum clean all && \
+  rpm --rebuilddb
+
+COPY package.json /app/package.json
+RUN npm --loglevel warn install --production
+COPY . /app
+
+CMD ["npm", "start"]
```

Comment:
Does this need a user to be specified (as per https://github.com/UKHomeOffice/firearms/blob/master/Dockerfile#L17) to not be running as root?